src/js/azdata/view/azDataGraph.js

```diff
@@ -200,3 +198,37 @@ azdataGraph.prototype.graphClickEventHandler = function(sender, event) {
         event.consume();
     }
 }
+
+/**
+ * Function: addZoomInListener
+ * 
+ * Adds the zoom in listener to the given element
+ * 
+ * Parameter:
+ * 
+ * element - The element to add the listener to.
+ */
+azdataGraph.prototype.addZoomInListener = function (element) {
```

Comment:
do you think this is a better design than just exposing a wrapper method that calls to `azdataGraph.prototype.zoomIn`, therefore letting the control host to wire-up their owning eventing?  One downside of this approach is it looks like we're limiting to only click.  What if the host wants to zoom based on keyboard events or double-click?

Reply:
No, you're right. The better design is to generalize the wrapper, so that any event type can trigger zooming and not just clicks.